engineering_blogs.opml

```diff
@@ -245,6 +245,7 @@
       <outline type="rss" text="Salesforce" title="Salesforce" xmlUrl="https://developer.salesforce.com/blogs/engineering/feed" htmlUrl="https://developer.salesforce.com/blogs/engineering/"/>
       <outline type="rss" text="Schibsted Tech Polska" title="Schibsted Tech Polska" xmlUrl="http://www.schibsted.pl/blog/feed/" htmlUrl="http://www.schibsted.pl/blog/"/>
       <outline type="rss" text="Scrunch" title="Scrunch" xmlUrl="https://tech.scrunch.com/blog/feeds/rss/" htmlUrl="https://tech.scrunch.com/blog/"/>
+      <outline type="rss" text="Scrapinghub" title="Scrapinghub" xmlUrl="https://blog.scrapinghub.com/comments/feed/" htmlUrl="https://blog.scrapinghub.com/"/>
```

Comment:
This links to the _comments_ feed. I'm pretty sure you want to link to the items feed: `https://blog.scrapinghub.com/feed/`.

Reply:
Good Catch. I fixed that.